ci/docker/runtime_functions.sh

```diff
@@ -2048,7 +2048,7 @@ cd_s3_publish() {
     if [ -z "${variant}" ]; then
         variant="cpu"
     fi
-    aws s3 cp ${filepath} s3://apache-mxnet/dist/${variant}/${filename} --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=43f628fab72838a4f0b929d7f1993b14411f4b0294b011261bc6bd3e950a6822
+    aws s3 cp ${filepath} s3://apache-mxnet/dist/python/${variant}/${filename} --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=43f628fab72838a4f0b929d7f1993b14411f4b0294b011261bc6bd3e950a6822
```

Comment:
How about the date? I see there are `2020-01-27` folders at s3://apache-mxnet/dist/ but `20200127` folders at `s3://apache-mxnet/dist/python/`.

Is any further change required to assimilate the date format among both pipelines?

Reply:
Yes, they will need to be copied. Those date-based folders will no longer be kept in the CD pipeline.

Reply:
The CodeBuild pipeline no longer publishes to that bucket, right?